practicesharp/Equation_Validator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+    class Equation_Validator
+    {
+        public string equation_s { get; private set; }
+
+        public Equation_Validator(string userinput)
+        {
+            equation_s = "";
+            if (Validate(userinput)) equation_s= userinput;
+        }
+        public bool Validate(string word)
+        {
+            try
+            {
+                const string allowed_letters = "1234567890+-*/";
+                foreach (char letter in word)
+                {
+                    if (allowed_letters.Contains(letter))
```

Comment:
You can use "Any" (LINQ function) to check if there is at least one instance of "letter" in "allowed_letters".
It will simplify code a bit.